core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,16 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput' &&
```

Comment:
@seanlip This is what I've had trouble with since the beginning. I understand the approach, but I've logged responsesService.getAnswerGroups() and the I don't see how to compare what's in getAnswerChoices with what's in get answerGroups, other than length.

<img width="342" alt="screen shot 2016-02-21 at 2 06 08 pm" src="https://cloud.githubusercontent.com/assets/10858542/13205073/664d2216-d8a4-11e5-9415-ba2f2448bec9.png">